src/mon/Monitor.cc

```diff
@@ -3244,12 +3270,14 @@ void Monitor::resend_routed_requests()
     if (mon == rank) {
       dout(10) << " requeue for self tid " << rr->tid << " " << *req << dendl;
       req->set_connection(rr->con);
-      retry.push_back(new C_RetryMessage(this, req));
+      MonOpRequestRef op = op_tracker.create_request<MonOpRequest>(req);
+      retry.push_back(new C_RetryMessage(this, op));
```

Comment:
hmm, won't this reset the op's lifecycle?  shouldn't we requeue the same op so that we see that it was originally proxied, then requeued locally?